yt/frontends/gdf/tests/conftest.py

```diff
@@ -0,0 +1,29 @@
+"""
+Title: conftest.py
+Purpose: Contains fixtures for loading data.
+"""
+
+
+# Test data
+sedov = "sedov/sedov_tst_0004.h5"
+
+
+# Test parameters. Format:
+# {test1: {param1 : [(val1, val2,...), (id1, id2,...)], param2 : ...}, test2: ...}
+test_params = {
+    "test_sedov_tunnel": {
+        "a": [(0, 1, 2), ("0", "1", "2")],
+        "d": [(None, ("sphere", ("max", (0.1, "unitary")))), ("None", "sphere")],
+        "w": [(None, "density"), ("None", "density")],
+        "f": [("density", "velocity_x"), ("density", "vx")],
```

Comment:
could the parameter names be clearer here?

Reply:
Sorry, yup! I changed the fixture names but forgot about these. I'll do that now.